lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/strategy/KbvStrategyParser.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.di.ipv.cri.kbv.api.strategy;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class KbvStrategyParser {
+    private final String strategy;
+
+    public KbvStrategyParser(String strategy) {
+        this.strategy = strategy;
+    }
+
+    public Strategy parse() {
+        List<Integer> numbers =
```

Comment:
This `List` can be confusing to read especially for people who don't understand the stream api. Maybe we can extract this out into it's own method and use a Pattern/Matcher combo, for example:

```
public static List<Integer> extractNumbers(String input) {
        List<String> numbers = new ArrayList<>();
        Pattern pattern = Pattern.compile("\\D+");
        Matcher matcher = pattern.matcher(input);
        while (matcher.find()) {
            numbers.add(Integer.parseInt(matcher.group()));
        }
        return numbers;
    }
```
I haven't tested the above.